todus/client.py

```diff
@@ -73,7 +84,9 @@ def _reserve_url(self, token: str, filesize: int) -> tuple:
                         (
                             "<iq i='"
                             + sid
-                            + "-3' t='get'><query xmlns='todus:purl' type='1' persistent='false' size='"
+                            + "-3' t='get'><query xmlns='todus:purl' type='"
+                            + str(file_type)
```

Comment:
```suggestion
                            + str(int(file_type))
```
como ahora es una enumeración, convertirla a string directamente no da un número sino el nombre del valor